guides/sdk-docs/runtime-libs/adding-asset-information-into-an-asset-home-extension.md

```diff
@@ -1,4 +1,4 @@
----
+e---
```

Comment:
What is this?

Reply:
no clue ... it looks like an e ... but who knows?